pkg/cosign/tuf/client.go

```diff
@@ -360,6 +383,12 @@ func embeddedLocalStore() (client.LocalStore, error) {
 	return local, nil
 }
 
+type signedMeta struct {
```

Comment:
Is there a struct defined in go-tuf we can use? For timestamp, can we use https://github.com/theupdateframework/go-tuf/blob/5b81b7e5604bfae2f0dcecd041e2e44f0128c992/data/types.go#L310?

Reply:
It's a private struct, not exported in go-tuf. I don't really want to condition on metadata type though, and all metadata defines these three.